### PR TITLE
Update backoffice-entry-point.md

### PR DESCRIPTION
**Pull request by HamDerAndrew**
Added some more details on how you make a umbraco-cloud.json (package manifest) for the backofficeEntryPoint.

## Description

Added some more details on how you make a umbraco-cloud.json (package manifest) for the backofficeEntryPoint.

## Type of suggestion

* [ ] Typo/grammar fix
* [X] Updated outdated content
* [ ] New content
* [ ] Updates related to a new version
* [ ] Other

### CODE REVIEW

File: 14/umbraco-cms/customizing/extending-overview/extension-types/backoffice-entry-point.md [hunk 1]
```diff
@@ -11,13 +11,18 @@ The `backofficeEntryPoint` extension is also the way to go if you want to load i
 
 The Entry Point manifest type is used to register an entry point for the backoffice. An entry point is a single JavaScript file that is loaded when the backoffice is initialized. This file can be used to do anything, this enables more complex logic to take place on startup.
 
-**Register an entry point in a JSON manifest**
+**Register an entry point in a umbraco-package.json manifest**
```
Comment (sofietoft) [inline 14/umbraco-cms/customizing/extending-overview/extension-types/backoffice-entry-point.md:14]:
```suggestion
**Register an entry point in the `umbraco-package.json` manifest**
```

File: 14/umbraco-cms/customizing/extending-overview/extension-types/backoffice-entry-point.md [hunk 2]
```diff
@@ -11,13 +11,18 @@ The `backofficeEntryPoint` extension is also the way to go if you want to load i
...
 
 ```typescript
```
Comment (sofietoft) [inline 14/umbraco-cms/customizing/extending-overview/extension-types/backoffice-entry-point.md:16]:
```suggestion
```json
```

**Review by sofietoft — COMMENTED** [review]
Looks great @HamDerAndrew ! 💪 

Just added a couple of minor suggestions.

File: 14/umbraco-cms/customizing/extending-overview/extension-types/backoffice-entry-point.md (hunk 2 above)
Comment (HamDerAndrew) [reply 14/umbraco-cms/customizing/extending-overview/extension-types/backoffice-entry-point.md:16]: Good one. Not sure why I didn't change the typescript 🤦

**Comment by HamDerAndrew** [comment]
Minor changes are looking good to me @sofietoft 😄

**Comment by sofietoft** [comment]
Thanks @HamDerAndrew ! 💪 

I'll get it merged.

**Pull request merged**